### PR TITLE
#91 Found a way to remove ###BREAK###

### LINKED ISSUE #91: [TASK] Content gathering
Find way to get rid of ###BREAK### when fetching content.

### PR DESCRIPTION
**Pull request by gordon81**
Resolves #91

### CODE REVIEW

File: Classes/ContentObject/ContentJsonContentObject.php [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+        do {
+            $records = $this->cObj->getRecords($conf['table'], $conf['select.']);
+            $cobjValue = [];
+            if (!empty($records)) {
+                $this->cObj->currentRecordTotal = count($records);
+                $this->getTimeTracker()->setTSlogMessage('NUMROWS: ' . count($records));
+
+                /** @var ContentObjectRenderer $cObj */
+                $cObj = GeneralUtility::makeInstance(ContentObjectRenderer::class);
```
[the remark this answers is not in the document]
Comment (gordon81) [reply Classes/ContentObject/ContentJsonContentObject.php:82]: AbstractContentObject has getContentObjectRenderer()

File: Classes/ContentObject/ContentJsonContentObject.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+ */
+class ContentJsonContentObject extends AbstractContentObject
+{
+    /**
+     * Rendering the cObject, CONTENT
+     *
+     * @param array $conf Array of TypoScript properties
+     * @return string Output
+     */
+    public function render($conf = [])
```
[the remark this answers is not in the document]
Comment (gordon81) [reply Classes/ContentObject/ContentJsonContentObject.php:35]: It is always a string. Json is a string.

[the remark this answers is not in the document]
Comment (gordon81) [reply Classes/ContentObject/ContentJsonContentObject.php:35]: if it's always a string, 
YES
we can set return type here, 
YES

also set $conf as array
YES you give the Configuration by typoscript and this is a Array

**Comment by turko** [comment]
This is great. This also solve the access right not being respected. Great work.

File: Classes/ContentObject/ContentJsonContentObject.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (tmotyl) [reply Classes/ContentObject/ContentJsonContentObject.php:35]: @gordon81 I think @lukaszuznanski has something like this in mind:
```
public function render($conf = []): string
```

**Pull request closed without merging**